src/pybel/io/mongodb.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""An exporter from PyBEL graphs to a local Mongo database."""
+
+import copy
+from typing import Any, List, Mapping, Tuple
+
+import pymongo
+from pymongo.collection import Collection
+
+import pybel
+from pybel.schema import is_valid_edge, is_valid_node
+
+__all__ = [
+    '_entity_to_dict',
+    'to_mongodb',
+    'find_nodes',
+    'get_edges_from_node',
+    'get_edges_from_criteria'
+]
+
+client = pymongo.MongoClient()
+
+
+def _handle_member(elem):
+    """Convert a member element from a complex or composite to a dictionary."""
+    # Convert each member element to a dictionary
+    new_elem = dict(elem)
+    # Remove the 'bel' and 'id' properties (which occur seemingly at random)
+    for prop in ('bel', 'id'):
+        if prop in new_elem.keys():
+            del new_elem[prop]
+    return new_elem
+
+
+def _entity_to_dict(entity: pybel.dsl.Entity) -> Mapping[str, Any]:
+    """Input a pybel Entity and return a dict representing it."""
+    new_node = dict(entity)
+    if new_node['function'] in ['Complex', 'Composite']:
+        new_node['members'] = list(map(_handle_member, new_node['members']))
+    return new_node
+
+
+def to_mongodb(graph: pybel.BELGraph, db_name: str, collection_name: str) -> Collection:
+    """Export the given BELGraph to a MongoDB.
+
+    In order to use this function, MongoDB must already be running locally.
+
+    :param graph: the graph to be exported
+    :param db_name: the name of the MongoDB to which the graph should be exported
+    :param collection_name: the name of the collection within the MongoDB where the graph will be stored.
+    :return: the collection that now stores the graph
+    """
+    # Access (or create) the specified database and collection
+    db = client[db_name]
```

Comment:
make the client a keyword argument that's optional:
```python
def to_mongodb(graph: pybel.BELGraph, db_name: str, collection_name: str, client: Optional[MongoClient] = None) -> Collection:
    if client is None:
        client = MongoClient()
    ...
```

This way, the user has the ability to specify stuff if they want, and we don't have to worry about the client as a global variable to the module (will help with testing)